### ISSUE TITLE
Image Ratio needed for featured image widget

### ISSUE THREAD

**Issue by amindiary** [pojome/elementor#5857]
- [x] I have searched for similar features requests in both open and closed tickets and cannot find a duplicate.
- [x] The feature is still missing in the latest stable version of Elementor ( Elementor Pro. )

Need item ratio option for featured image or any image element.
However I added it with this code
https://gist.github.com/amindiary/cb51291dc546dd70a832d8e988b0dae1
 but  its dirty and need to be done in core.

![image](https://user-images.githubusercontent.com/2544046/46499815-ccccfa80-c829-11e8-88a0-5a56144f44b9.png)

**Comment by bainternet** [comment]
Currently, this feature is not supported, but we will look into it in the future if we see a large demand for it.

You are much better off adding an image size via [WordPress API](https://developer.wordpress.org/reference/functions/add_image_size/) and letting WordPress take care of the resizing/cropping.


Thanks for the feedback.

**Comment by amindiary** [comment]
@bainternet well, if its not supported, why it is available on posts widget?

**Comment by dimkasta** [comment]
It would very useful to be able to set this, as featured image is often used on a template level. It is already supported on a page/post level, but it is far from efficient or even practical to go through hundreds of posts just to change or fine tune the featured image ratio on different devices

**Comment by mgustavs** [comment]
I've been breaking my brain over this for days here. I need pictures that change on hover (to another pic) with links. My client needs these pictures to be visible in the original ration on any device - 320 up to 1920 wide - 4/9. I have sniffed every widget I know on Pro and I can't achieve this! If I just put a photo, there is a ration and I can add link, but no other picture on hover. Whatever else I choose - call to action | flip-box | column (with added link) - I set the min height or I don't - they all do have different aspect ratios on different screens. They all tweak the photo to their needs (or display halfway) and there is no way to tell all those widgets to have same ratio as the added photo.

To summarize - either full on equipped picture widget that can change on hover or those other widgets adjusting their height by added photos aspect ratio would be great!

**Comment by TheRollingPanda** [comment]
+1

There is still no way to control the image ratio or cropping of the featured image widget. If I upload a good image ratio for desktop, it looks too small on mobile, and vice versa.

**Comment by AlexeySvidentsov** [comment]
Feature still required

**Comment by Lovely-Fellow** [comment]
Needed for me, too

**Comment by igorchernenko92** [comment]
needed for me

**Comment by glampe** [comment]
Need for loop grids

**Comment by kitka** [comment]
I would love to have this too
